ansible_collections/azureredhatopenshift/cluster/tasks/create_aro_cluster.yaml

```diff
@@ -726,7 +726,9 @@
 
 - name: create_aro_cluster | Register oc_arch
   ansible.builtin.set_fact:
-    oc_arch: "{{ oc_arch_map[localhost_facts.ansible_facts['ansible_architecture']] }}"
+    oc_arch: >-
+      {{ oc_arch_map[localhost_facts.ansible_facts['ansible_architecture']] if delegation == "localhost"
+      else oc_arch_map[jumphost_facts.ansible_facts['ansible_architecture']] }}
```

Comment:
Can we do something like `oc_arch_map[hostvars[delegation].ansible_architecture]` instead?

Reply:
Since my suggestion will need further changes, let's save it for a later PR